content/en/blog/_posts/XX-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: blog
+title: "Kubernetes 1.33: User Namespaces enabled by default!"
+date: XXXX-XX-XX
```

Comment:
While this blog is about the v1.33 release, we will limit to one blog published per day. This means it is likely that this blog will be planned on a later date, and the schedule is yet to be fully decided. (It is highly dependent on other blog reviews completed in time)

If you can simply add `draft: true` in the front matter, it would allow us to merge the blog without actually publishing them to the website. Once the blog content is finalised and merged, the Release Comms team can take over the rest of the publication setup.

```suggestion
date: 2025-04-23
draft: true
```

You can find more in the blog guideline [here](https://kubernetes.io/docs/contribute/blog/article-submission/#article-scheduling)! 👍